docs/configuration/index.md

```diff
@@ -774,6 +774,12 @@ The information about releases [is retrieved from Bodhi](https://bodhi.fedorapro
 cache and required availability on Copr, it might take a while to get the
 newest state.
 
+#### sync_test_job_statuses_with_builds
+
+(*bool*) When set to `true` (the default), test job statuses are updated together with their corresponding
+build job statuses. When set to `false`, while tests jobs are waiting for their corresponding build jobs
```

Comment:
I would maybe just add some example of those, e.g. `SRPM build is in progress..`